index.js

```diff
@@ -44,4 +44,9 @@ function patternImporter (options) {
 
 }
 
-module.exports = patternImporter;
+module.exports = exports = patternImporter;
```

Comment:
as far as I know ```exports``` is alias of  ```module.exports```, ```module.exports = patternImporter ``` should be enough.
Also, I think it makes sense to export utils clearly as a property of object:
```js
module.exports = {
  patternImporter: patternImporter,
  utils: utils
}
```
or I saw some modules do even like this:
```js 
module.exports = function () {
  return function (export) {
    if (export === 'utils') {
      // can be received by require('patternImporter')('utils')
      return require('./lib/utils');
    }
    if (export === 'streamHandler') {
      // can be received by require('patternImporter')('streamHandler')
      return patternImporter;
    }
  } 
}
```